docs/docs/getting-started/first-steps.md

```diff
@@ -95,6 +95,27 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
     Your Constellation cluster was created successfully.
     ```
 
+   :::info
+
+   On Azure, you may receive the following error when running with limited IAM permissions:
```

Comment:
> I am not 100% happy with the large troubleshooting block, but as far as I know we don't have any other troubleshooting section

We have Troubleshooting under Workflows. You may put it there if you prefer. I'm fine with both.